src/nav/nrql.yml

```diff
@@ -29,40 +29,40 @@ pages:
         path: /docs/nrql/nrql-references/nrql-group-results-across-time
       - title: NRQL math
         path: /docs/nrql/nrql-references/nrql-math-using-select
-  - title: Query tools
+  - title: NRQL tools
     pages:
       - title: New Relic query builder
         path: /docs/query-data/nrql-new-relic-query-language/query-tools/chart-builder-create-query-visualizations
       - title: NerdGraph (GraphQL API)
         path: /docs/query-data/nrql-new-relic-query-language/query-tools/nrql-api-queries
-  - title:  Explore NRQL
+  - title: NRQL querying
```

Comment:
`Query NRQL` reads better, even if it eschews the NRQL-as-adjective-noun structure of the other titles.